src/si/energy.rs

```diff
@@ -56,6 +56,9 @@ quantity! {
         @decaelectronvolt: 1.602_176_634_E-18; "daeV", "decaelectronvolt", "decaelectronvolts";
         @electronvolt: 1.602_176_634_E-19; "eV", "electronvolt", "electronvolts";
 
+        // Atomic unit of energy (Hartree energy)
+        @hartree: 4.359_744_722_2071_E-18; "Ha", "hartree", "hartrees";
```

Comment:
`Eₕ` seems more common than `Ha`. Thoughts?

```suggestion
        /// Atomic unit of energy (Hartree energy).
        @hartree: 4.359_744_722_207_1_E-18; "Eₕ", "hartree", "hartrees";
```